src/components/item/ItemPreviewCard.vue

```diff
@@ -11,27 +11,53 @@
     :blank-image-height="280"
     :variant="variant"
     :lazy="lazy"
+    :sub-title="subTitle"
   >
     <template #buttons>
-      <RecommendationButtons
-        v-if="enableAcceptRecommendation || enableRejectRecommendation"
-        :identifier="identifier"
-        :enable-accept-button="enableAcceptRecommendation"
-        :enable-reject-button="enableRejectRecommendation"
-      />
-      <UserButtons
-        v-else
-        :identifier="identifier"
-        :show-pins="showPins"
-        @like="$emit('like', identifier)"
-        @unlike="$emit('unlike', identifier)"
-      />
+      <div
+        v-if="variant === 'list'"
+        class="data-and-buttons-wrapper d-flex flex-wrap"
+      >
+        <RightsStatement
```

Comment:
This is not a button, neither is the type text that follows, so it doesn't seem quite right to include it in a slot named "buttons". Perhaps we should rename the "buttons" slot to indicate that its content will be rendered over the image, e.g. "image", then add a new slot named "footer" for the location used by the text + buttons in the updated list view.